packages/playwright-core/src/server/webkit/webkit.ts

```diff
@@ -98,3 +108,8 @@ export class WebKit extends BrowserType {
     return webkitArguments;
   }
 }
+
+export async function translatePathToWSL(path: string): Promise<string> {
```

Comment:
As discussed before, perhaps do this once, save somewhere on `WKBrowser` and then just `path.join()` when needed?